server/webapp/WEB-INF/rails.new/spec/webpack/views/dashboard/pipeline_widget_spec.js

```diff
@@ -22,12 +22,13 @@ describe("Dashboard Pipeline Widget", () => {
 
   const PipelineWidget = require("views/dashboard/pipeline_widget");
   const Pipelines      = require('models/dashboard/pipelines');
+  const Dashboard      = require('models/dashboard/dashboard');
   const DashboardVM    = require("views/dashboard/models/dashboard_view_model");
   const Modal          = require('views/shared/new_modal');
   const pipelineName   = 'up42';
 
-
-  let $root, root, dashboardViewModel, pipelinesJson, pipeline, doCancelPolling, doRefreshImmediately;
+  let $root, root, dashboardViewModel, dashboard, pipelinesJson, dashboardJSON, pipeline, doCancelPolling,
+      doRefreshImmediately;
```

Comment:
define it in single line?

Reply:
That is intellij formatting the code because the line was getting long.